### PR TITLE
Mercenary Adjustments 2.whatever

### PR DESCRIPTION
**Pull request by ProbablyCarl**
## About The Pull Request
Adjusts the mercenary guild's rooftop overhang, so it can't be climbed onto as easily, if at all, from prior locations. Without building upwards, anyhow.

Adjusts the Kommandant's PQ requirement to be 6 and gives him some better equipment to differentiate him easier. Blacksteel chausses and blacksteel boots. Also gives him a chainmail shirt instead of the poofy one, and a chainmail coif.

Removes one slot from Grenzel, just to account for the Kommandant, even though we'd never have more than ten slots in total. Just because it was bugging me.

## Why It's Good For The Game
Kommandant is intended to be a better equipped role, with a higher standard of play requirement. This reflects that, by upping the PQ requirement and significantly adjusting their equipment.

As for the rooftop, it's just because I like to climb on it, smash windows and flee. The players didn't like this. :(

### CODE REVIEW

**Comment by katsukai** [comment]
Since when was there a mary sue Grenzel OC in game

File: code/modules/jobs/job_types/roguetown/mercenaries/types/grenzelhoft_kommandant.dm [hunk 1]
```diff
@@ -50,11 +50,9 @@
 		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/maces, 2, TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/bows, 1, TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 4, TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/swords, 5, TRUE)
```
Comment (Gyrandola) [inline code/modules/jobs/job_types/roguetown/mercenaries/types/grenzelhoft_kommandant.dm:55]: powercreep

Comment (Nakalon) [reply code/modules/jobs/job_types/roguetown/mercenaries/types/grenzelhoft_kommandant.dm:55]: 😡

Comment (Nakalon) [reply code/modules/jobs/job_types/roguetown/mercenaries/types/grenzelhoft_kommandant.dm:55]: I'm of the opinion that no skill should be superior to 4 round start.

**Comment by Gyrandola** [comment]
Im worried grenzel mercs are getting a bit too strong but we'll see.

**Review by Gyrandola — APPROVED** [review]

**Pull request merged**